project/views.py

```diff
@@ -128,11 +167,10 @@ def parse_labels(labels):
     for label in labels:
 
         if str(label["description"]).lower() == LABEL_MENTOR:  # Parsing Mentor
-            # print("dfs ",label["name"])
             mentor = parse_mentor(label["name"])
 
         if str(label["description"]).lower() == LABEL_LEVEL:  # Parsing Level
-            level, points = parse_level(label["name"])  # Fetching Level and it's default point
+            level, points = parse_level(label["name"])  # Fetching Level and its default point
```

Comment:
it's its which is correct ?

Reply:
Sir I asked GPT it says its will be correct